src/sentry/options/defaults.py

```diff
@@ -623,6 +623,7 @@
 register("performance.issues.n_plus_one_db.problem-creation", default=0.0)
 register("performance.issues.n_plus_one_db_ext.problem-creation", default=0.0)
 register("performance.issues.file_io_main_thread.problem-creation", default=0.0)
+register("performance.issues.db_main_thread.problem-creation", default=0.0)
```

Comment:
This isn't no longer required, we now auto register the options in getsentry.
see https://github.com/getsentry/getsentry/pull/9794

This will automatically register the following admin options


performance.issues.<my_issue>.ea-rollout 
performance.issues.<my_issue>.flag_disabled 
performance.issues.<my_issue>.ga-rollout
performance.issues.<my_issue>.la-rollout

Reply:
Will get even better soon, once perf issues are on the issue platform you won't even need to register a feature handler: https://github.com/getsentry/getsentry/blob/8635687247ec664fdd68ddcb40a533dfe7c2d0f6/getsentry/features.py#L964-L1034

![Screen Shot 2023-03-17 at 7 22 09 AM](https://user-images.githubusercontent.com/6288560/225743848-d8d18fbb-cdfe-4ec8-a990-21459a021cc6.png)
![Screen Shot 2023-03-17 at 7 22 21 AM](https://user-images.githubusercontent.com/6288560/225743858-99bf4b7c-7a2a-4ad9-ab7a-1fbcd26ebdef.png)

Reply:
Is it OK to just leave it as-is for now? Or are we looking to standardize all the feature handling on these issues?

Reply:
Neat info on the issue platform!

We were planning to switch to the new handlers on all detectors for standardization. If were all going to move to issue platform eventually then I suppose it doesn't matter much, I'm just not sure on the timeline on when we'll move.

I would probably do the new feature handler anyways because imo it's much easier, but will leave the decision to you!